src/algorithms/tracking/TrackParamTruthInit.cc

```diff
@@ -72,6 +72,12 @@ eicrecon::TrackParameters *eicrecon::TrackParamTruthInit::produce(const edm4hep:
         return nullptr;
     }
 
+    // modify initial momentum to avoid bleeding truth to results when fit fails
+    // this picks uniformly between [1-eps,1,1+eps] times true momentum, then smeared
+    const auto pinit = pmag 
+                     * (1.0 + m_momentum_split * m_uniformIntDist(generator))
```

Comment:
It is suggested to use config POD for parameters such as m_momentum_split

Reply:
Out of scope.